containers/unit-tests/run.sh

```diff
@@ -4,6 +4,7 @@ set -o pipefail
 set -eux
 
 export LANG=C.UTF-8
+export MAKEVARS="-j $(nproc)"
```

Comment:
Isn't that `MAKEFLAGS`?  I couldn't find any documentation for `MAKEVARS` and no reference to it in our Makefiles.

Reply:
Yeah, it is `MAKEFLAGS`: https://www.gnu.org/software/make/manual/html_node/Options_002fRecursion.html

Reply:
Whoa, right! I've had `MAKEVARS` in my .bashrc for years, apparently it never worked.. Fixed, thanks!